### DISCUSSION TITLE
Using 2D weights for 3D models

### DISCUSSION THREAD

**Discussion by CJ2588** [Project-MONAI/tutorials#1846] [Q&A]
I was wondering if Monai has an inbuilt function/method to inflate the 2D model pre-trained weights for their 3D counterparts. If their isn't would building a custom function to accomplish this be advisable?

**Comment by KumoLiu** [comment]
Hi @CJ2588, as far as I remember, we don't seem to have this feature yet. CC @ericspod in case I missed something.

**Reply by ericspod** [reply]
I don't think we do either, no. Unless you have references indicating how to do this and where it's useful, I'm not sure how much of a good idea it is to do this. If there's other research out there on this topic we can look at it and see what to do about integration.
